test/flickr.auth.checkToken.js

```diff
@@ -14,7 +14,9 @@ describe('flickr.auth.checkToken', function () {
 	});
 
 	it('returns a Request instance', function () {
-		var req = flickr.auth.checkToken({ auth_token: '_' });
+		var req = flickr.auth.checkToken({
+			auth_token: '_'
+		});
```

Comment:
Knowing it's generated code and not code I wrote is what helps me sleep at night.